### ISSUE TITLE
Google reCAPTCHA v2 Fails to Function on Customer Registration Page

### ISSUE THREAD

**Issue by lanaworks** [nenes25/eicaptcha#261]
### Check if your issue exists

- [X] I confirm that i've search that my issue does not already exists

### Your prestashop version

8.1.x

### Eicaptcha version

2.4.x

### Do you use a specific theme ?

Panda Theme (v286) with child theme 

### Describe the bug

The Google reCAPTCHA v2 module is not functioning as expected on the customer registration page. While the reCAPTCHA block is displayed correctly, it is possible to proceed with account creation without actually checking the reCAPTCHA box.

### Additional context

**Environments Where Issue Occurs:**
- Contact form page: Working correctly
- Shopping cart process: Working correctly
- Customer registration page: Not working correctly

**Expected Behavior:**
On the customer registration page, users should be required to check the Google reCAPTCHA v2 box before they can successfully create an account.

**Actual Behavior:**
The Google reCAPTCHA v2 block appears on the customer registration page, but it does not prevent account creation when left unchecked. This allows users to proceed without completing the reCAPTCHA challenge.

**Comment by nenes25** [comment]
Hello,

This will be fixed in the next version.
It's almost ready but i need to test it before the launch.
Feel free to grab the code of the it, in the dev branch.

Regards,

**Comment by lanaworks** [comment]
Hi,

Thank you for your prompt response and for the efforts you've made in addressing the issue. I wanted to inform you that I've tested the code from the development branch as you suggested, and I am pleased to report that the problem with the Google reCAPTCHA v2 on the customer registration page has been resolved. The reCAPTCHA now functions as expected, requiring verification before account creation can proceed.

I appreciate the swift action taken to rectify this issue. Your commitment to maintaining and improving the module is truly commendable. Thank you once again for your diligent work and for providing a timely solution.

Best :)
